fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/dto/DateParmVal.java

```diff
@@ -52,46 +50,16 @@ public void setValueDateEnd(Timestamp valueDateEnd) {
         this.valueDateEnd = valueDateEnd;
     }
 
-    public void setName(String name) {
-        this.name = name;
-    }
-
-    public String getName() {
-        return name;
-    }
-
-    public String getResourceType() {
-        return resourceType;
-    }
-
-    public void setResourceType(String resourceType) {
-        this.resourceType = resourceType;
-    }
-
     /**
      * We know our type, so we can call the correct method on the visitor
      */
     public void accept(ExtractedParameterValueVisitor visitor) throws FHIRPersistenceException {
         visitor.visit(this);
     }
 
-    /**
-     * @return the base
-     */
-    public String getBase() {
-        return base;
-    }
-
-    /**
-     * @param base the base to set
-     */
-    public void setBase(String base) {
-        this.base = base;
-    }
-
     @Override
     public String toString() {
-        return "DateParmVal [resourceType=" + resourceType + ", name=" + name
+        return "DateParmVal [resourceType=" + getResourceType() + ", name=" + getName()
```

Comment:
Use getBase if base is private.

Reply:
base member removed (not used), and getBase() used in the toString as suggested